KishonStClair-Fatima.js

```diff
@@ -0,0 +1,83 @@
+// Question 1
+//Write a function calculateCube that takes a single number and prints the volume of a cube made from that number.
+
+function calculateCube(number) {
+  
+    console.log(number * number * number)
+}
+calculateCube(5)
+
+
+// Question 2
+//  write a function with an parameter for a string
+function isAVowel(str) {
+    // write a agrument for the length of the string and determine its a vowel
+    let count = 0
+    const vowels = "abpurtRWKO"
+    for(var i = 0; i < str.length; i++){
+    console.log(vowels.indexOf(str[i]) > -1)
+    }
+    
+  } 
+  
+  isAVowel("a")
+
+
+  // Question 3
+  function getTwoLengths(str) {
+  const name = "Hank";
+  const lastname = "Hippopopalous";
+  console.log([name.length, lastname.length])
+}
+getTwoLengths()
```

Comment:
nice work with this! the only thing I'm curious about is, why do you have a parameter in the function if you never use it? I think it would be good to have two parameters, `str1` and `str2`, and set `name = str1` and `lastname = str2`. that way, it works for any 2 words, not just the ones you wrote